cadence/contracts/bridge/FlowEVMBridge.cdc

```diff
@@ -0,0 +1,405 @@
+import "FungibleToken"
+import "NonFungibleToken"
+import "FlowToken"
+
+import "EVM"
+
+import "ICrossVM"
+import "IFlowEVMNFTBridge"
+import "FlowEVMBridgeConfig"
+import "FlowEVMBridgeUtils"
+import "IEVMBridgeNFTLocker"
+import "FlowEVMBridgeTemplates"
+
+/// The FlowEVMBridge contract is the main entrypoint for bridging NFT & FT assets between Flow & FlowEVM.
+///
+/// Before bridging, be sure to onboard the asset type which will configure the bridge to handle the asset. From there,
+/// the asset can be bridged between VMs using the public entrypoints below, the only distinctions being the type of
+/// asset being bridged (NFT vs FT) and the direction of bridging (to or from EVM).
+///
+/// See also:
+/// - Code in context: https://github.com/onflow/flow-evm-bridge
+/// - FLIP #237: https://github.com/onflow/flips/pull/233
```

Comment:
AFAIK we're numbering FLIPs by their issues, and #237 corresponds to https://github.com/onflow/flips/issues/237. The FLIP PR is mentioned in these comments since the FLIP isn't yet merged into onflow/flips:main, so I don't yet have a link to the .md file on main